src/test/java/pl/nogacz/chess/board/UndoTest.java

```diff
@@ -0,0 +1,31 @@
+package pl.nogacz.chess.board;
```

Comment:
Testlerin çok sınırlı kalmış. Test konusuna biraz daha kafa yor ve değişikliklerin ile ilgili manalı testler ekle. Eğer test yazmakta zorlanıyorsan o zaman demekki değişikliklerini yeteri kadar modüler metotlar halinde yapmamışsın demektir. Eğer yaptıysan o bireysel metotlar için testler yapabilirsin. Ama tüm değişikliği kocaman bir blok olarak yazdıysan o kodu test etmen çok mümkün olmaz.